cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -99,6 +99,13 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
+    {# Google Tag Manager #}
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Should the `<script>` tags have `async` set to true? I'm not sure if that would delay the start of analytics tracking.